Fix: Discrepancy in validators minimum requirements (#351)
polkadot-developers/polkadot-docs: infrastructure/running-a-validator/onboarding-and-offboarding/set-up-validator.md

## Initial Setup

> Before you can begin running your validator, you'll need to configure your server environment to meet the operational and security standards required for validating.

Reviewer: ```suggestion
Before running your validator, you must configure your server environment to meet the operational and security standards required for validating.
```

grammarly stuff
